std_msgs/Header header
float32[] global_cx
float32[] global_cy
float32[] global_cyaw
float32[] global_ck
float32[] global_sp

